src/app/styles/css/typography.css

```diff
@@ -1,3 +1,12 @@
+:root {
+  font-family: Inter, sans-serif;
+  font-feature-settings: 'liga' 1, 'calt' 1; /* fix for Chrome */
```

Comment:
There are some great disambiguation features of this font, so we take advantage of that if the browser supports it